packages/blade/src/components/ActionList/StyledActionListItem.web.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+import type { StyledActionListItemProps } from './getBaseActionListItemStyles';
+import { getBaseActionListItemStyles } from './getBaseActionListItemStyles';
+import Box from '~components/Box';
+
+const StyledActionListItem = styled(Box)<StyledActionListItemProps>((props) => {
+  return {
+    ...getBaseActionListItemStyles(props),
+    // Web-specific styles
+    '&:hover': {
+      backgroundColor:
+        props.intent === 'negative'
+          ? props.theme.colors.feedback.background.negative.lowContrast
+          : props.theme.colors.brand.gray.a50.lowContrast,
+    },
+    '&.active-focus': {
```

Comment:
Where do get or set this .active-focus class?

Reply:
https://github.com/razorpay/blade/blob/736d33458bdd9a703651d18df2c7b73a61089b68/packages/blade/src/components/ActionList/ActionListItem.tsx#L155

Here